### PR TITLE
#12 add backup and restore utils bash script

### LINKED ISSUE #12: add utils to backup and restore content
There is a need to provide a backup and restore utility for existing Spyderisk deployments. In particular we want to store users and models, possibly KB.

### PR DESCRIPTION
**Pull request by panositi**

### CODE REVIEW

File: .env_adaptor
Comment (scp93ch) [inline .env_adaptor:1]: looks like this was committed by mistake

File: .env
Comment (scp93ch) [inline .env:1]: looks like this was committed by mistake

File: docker-compose.yml
Comment (scp93ch) [inline docker-compose.yml:1]: should this be committed?

File: .env
Comment (panositi) [reply .env:1]: it includes EULA params

File: docker-compose.yml
Comment (panositi) [reply docker-compose.yml:1]: it has EULA params too

File: .env
Comment (scp93ch) [reply .env:1]: Yes I can see what it includes, but this has nothing to do with adding a backup script. There's nothing wrong with the short EULA_HTML text that you've replaced.

File: docker-compose.yml
Comment (scp93ch) [reply docker-compose.yml:1]: I think we're looking at different things. I see the addition of an openvas bind volume that we used in a particular deployment but is not generally required.

Comment (panositi) [reply docker-compose.yml:1]: The adaptor risk calculation will fail if there is no openvas volume mounted. These changes are not the core part of the backup and restore utils tho.

**Comment by scp93ch** [comment]
@panositi please can you remove the changes that are not related to the backup script and also add a reference to the backup script existing in the README? Then I can approve the merge.

Then open a separate issue about the openvas bind mount and we can work out what to do with that there.

Thanks.

**Comment by scp93ch** [comment]
The backup script itself looks great by the way!

**Comment by scp93ch** [comment]
@panositi can you check the changes I just committed? If you agree then I'll merge.

**Review by panositi — COMMENTED** [review]
Changes are accepted

**Pull request merged**